stdlib/3/urllib/response.pyi

```diff
@@ -1,12 +1,14 @@
 # private module, we only expose what's needed
 
-from typing import BinaryIO, Iterable, List, Mapping, Optional, Type, TypeVar
+from typing import BinaryIO, Callable, Iterable, List, Mapping, Optional, Type, TypeVar
```

Comment:
```suggestion
from typing import BinaryIO, Callable, Iterable, List, Mapping, Optional, Type, TypeVar, Tuple, Any
```

Reply:
The CI errors seem to be due to these imports missing

Reply:
Thanks!